assets/src/edit-story/components/library/libraryProvider.js

```diff
@@ -44,18 +44,21 @@ function LibraryProvider({ children }) {
   const [tab, setTab] = useState(MEDIA);
   const insertElement = useInsertElement();
 
-  const state = {
-    state: {
-      tab,
-    },
-    actions: {
-      setTab,
-      insertElement,
-    },
-    data: {
-      tabs: TABS,
-    },
-  };
+  const state = useMemo(
+    () => ({
+      state: {
+        tab,
+      },
+      actions: {
+        setTab,
+        insertElement,
+      },
+      data: {
+        tabs: TABS,
+      },
+    }),
+    [tab, setTab, insertElement]
```

Comment:
I'm surprised the `setTab` is here since it's a local `useState`. Doesn't hurt, but curious if linter gets confused here maybe?

Reply:
Looks like the linter didn't complain, so it was more that I was confused here :)
Updated.